glide.yaml

```diff
@@ -30,27 +30,31 @@ import:
   - transport/tchannel
 - package: gopkg.in/yaml.v2
   version: ^2.2.1
+- package: google.golang.org/grpc
+  version: ^1.22.0
+- package: github.com/prometheus/procfs
```

Comment:
This package is somehow in the dep graph and does not have a major version. It's included as part of the prometheus client.